### ISSUE TITLE
OSGEO Installer workflow

### ISSUE THREAD

**Issue by MattReimer** [Riverscapes/QRAVEPlugin#90]
When the user installs QGIS through the OSGEO installer they are able to pick and choose which python modules they can install. 

The user should NOT use pip to install these modules. They should navigate to the `C:\OSGEO4W\bin` folder (or wherever they installed it) and use the `osgeo12-setup.exe` tool to install the modules.

There are some simple mitigation steps we can do to make sure OSGEO installer users aren't left out in the cold. 

1. Let's make a list of recommended python modules across all our tools (lxml, scipy, shapely etc) and document it on our website for OSGEO users so they can go through and install them all before working with any of our plugins
2. look into better import error handling inside QRave so that the user receives better errors and instructions inside QRave when things go wrong.

So far, the list of modules that we needed to install is as follows:

```
lxml
scipy
shapely
```

**Comment by MattReimer** [comment]
Note: Re: #86 I installed QGIS using the OSGEO4W installer (QGIS and QGIS-LTR) and it installed all the right modules.

This ticket only applies to non-default, custom installs

**Comment by joewheaton** [comment]
@MattReimer I removed scipy and shapely as those were not needed to get QRAVE working. They were needed to get QRiS working. https://github.com/Riverscapes/QRiS/issues/152

**Comment by MattReimer** [comment]
I think, given that my standard OSGEO4W install did the right thing this ticket's importance drops almost to zero. We can handle the 1% custom cases with a little documentation and some phone calls.

**Comment by joewheaton** [comment]
Agreed. Maybe we update the QGIS installation instructions to discourage using OSGEO4W, and then have a buried sub-section or page showing that if someone does that form, and doesn't select the right stuff, what to do to fix it.

Can we still improve the error message it throws?

**Comment by MattReimer** [comment]
For now I'd say no. Too many variations on what could be thrown with a custom installer. Might be worth an investigation task at some point but right now we have bigger 🐟 to 🔥
